spec/rails_helper.rb

```diff
@@ -90,3 +90,10 @@
     end
   end
 end
+
+
+describe "kill", :focus do
```

Comment:
I had it run a single test for faster deployments. A horrible hack for the new year.